packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -131,6 +137,7 @@ export const chatMessageClassName = 'ui-chat__message';
 export const chatMessageSlotClassNames: ChatMessageSlotClassNames = {
   actionMenu: `${chatMessageClassName}__actions`,
   author: `${chatMessageClassName}__author`,
+  header: `${chatMessageClassName}__header`,
```

Comment:
We have a dedicated component now, we don't need this

Reply:
removed